sellercenter-attributes/src/main/java/com/gfg/sellercenter/attributes/entity/AttributeWithOption.java

```diff
@@ -0,0 +1,18 @@
+package com.gfg.sellercenter.attributes.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import java.util.Map;
+
+@AllArgsConstructor
+@Getter
+@EqualsAndHashCode
+public class AttributeWithOption {
+
+    private final int idCatalogAttribute;
+
+    private final String label;
+
+    private final Map<Integer, AttributeOption> attributeOptions;
```

Comment:
I'm not sure I understand how one single product for one attribute can have multiple options? Or is it result of combination of a same attribute from different product?

Reply:
e.g. color can have multiple options.

Reply:
shame on me, thanks, now I get this part (don't know how could I forget about multivalues attributes)